airlock/business_logic.py

```diff
@@ -54,9 +54,9 @@ class RequestStatus(Enum):
 
     def description(self):
         if self == RequestStatus.PARTIALLY_REVIEWED:
-            return "1 REVIEW COMPLETE"
+            return "ONE REVIEW SUBMITTED"
```

Comment:
drive-by fix because "1 review complete" scanned oddly to me in the request status pills

Reply:
> I would go with AP style & use words for 0-9 and digits for 10+

https://bennettoxford.slack.com/archives/C31C8R5K2/p1714145241135439